index.html

```diff
@@ -167,8 +175,8 @@ <h3>Example</h3>
       "given_name": "SUSAN",
       "portrait": "/9j/4AAQSkZJRgABAQEAkACQA...gcdgck5HtRRSClooooP/2Q==",
       "birth_date": "1998-08-28",
-      "issue_date": "2018-01-15T10:00:00.0000000-07:00",
-      "expiry_date": "2022-08-27T12:00:00.0000000-06:00",
+      "issue_date": "2023-01-15T10:00:00.0000000-07:00",
+      "expiry_date": "2028-08-27T12:00:00.0000000-06:00",
```

Comment:
Seems these would be ok without sub-second precision:
```suggestion
      "issue_date": "2023-01-15T10:00:00-07:00",
      "expiry_date": "2028-08-27T12:00:00-06:00",
```